recipes/hunspell/all/conanfile.py

```diff
@@ -25,41 +28,69 @@ class HunspellConan(ConanFile):
         "shared": False,
         "fPIC": True,
     }
-    generators = "cmake"
-    # FIXME: Remove once the pending upstream PR for CMake support is merged
-    exports_sources = "CMakeLists.txt"
-    no_copy_source = True
+
+    @property
+    def _minimum_cpp_standard(self):
+        return 11
+
+    @property
+    def _minimum_compilers_version(self):
+        return {
+            "gcc": "7",
```

Comment:
```suggestion
            "gcc": "7",
```
What about other compilers? GCC-7 Are you sure? C++11 is supported since GCC-4.7 (partially)

Reply:
I only updated this list due to the configurations that were failing for the conan 1 CI builds. I can't comment on any other compiler nor standard (I am _just_ a user of the library). 
I quickly tried to get to the CI failure but could not find it.

Reply:
I might have made an mistake with 7 (vs 4.7) checking that as well

Reply:
From the [last build](https://c3i.jfrog.io/c3i/misc/summary.html?json=https://c3i.jfrog.io/c3i/misc/logs/pr/18829/13-linux-gcc/hunspell/1.7.2//summary.json) there is a failure in 5. 
7 Is then the lowest next version available in CI thus used it. Reverting back to 7